### PR TITLE
Class22-24-body-component

### PR DESCRIPTION
**Pull request by Hailegebriel22**
# Description

Main body component

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/body/body.component.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import './body.style.css';
+import WomenAccessories from '../../assets/women-accessories.png';
```
Comment (Hanouj) [inline packages/client/src/components/body/body.component.js:3]: Hi Haile I can see that you put the images in client/src/assets and then you import them here. 
an easier way to do it, is to use the public folder in the client and then in assets you'd create a folder for ur images 
then you can access it from any folder you are without importing them like that (src="/public/assets/foldername/glass.png")

**Review by Hanouj — DISMISSED** [review]
over all good work Haile

**Review by shpomp — APPROVED** [review]
❗ Fix capitalisation:
image

**Comment by shpomp** [comment]
> ❗ Fix capitalisation:
> image
> 

@Hailegebriel22 the folders and files should be edited 🙏

**Review by shpomp — CHANGES_REQUESTED** [review]
> ❗ Fix capitalisation:
> image
> 

@Hailegebriel22 the folders and files should be edited 🙏

**Review by Hanouj — APPROVED** [review]

**Review by Hanouj — APPROVED** [review]

**Review by shpomp — APPROVED** [review]

**Pull request merged**